frontend/src/pages/Landing/index.js

```diff
@@ -0,0 +1,331 @@
+import React, { useState, useEffect } from 'react';
+import { Tag, Table, Alert } from '@trussworks/react-uswds';
+import { Helmet } from 'react-helmet';
+import moment from 'moment';
+import { Link } from 'react-router-dom';
+
+import Container from '../../components/Container';
+import { getReports } from '../../fetchers/activityReports';
+import 'uswds/dist/css/uswds.css';
+import '@trussworks/react-uswds/lib/index.css';
+import './index.css';
+
+export const activityReportId = (id, regionId) => `R${
+  regionId < 10
+    ? `0${regionId}`
+    : regionId
+}-${id <= 999999 ? `00000${id}`.slice(-6) : id}`;
+
+export const getValue = (item, sortConfig) => {
+  const keys = sortConfig.key.split('.');
+
+  // const getValueForItem = (item) =>
+  return keys.reduce((object, key) => {
+    const result = (object || {})[key];
+    if (key === 'activityRecipients' || key === 'collaborators') {
+      return result[0].name;
```

Comment:
When sorting by collaborators I get the following error here `TypeError: Cannot read property 'name' of undefined`. I had reports that did and did not have collaborators

Reply:
Good catch. I refactored name, but didn't update it here.

Reply:
:+1: Don't see the error with the newest changes